AmplifyPlugins/DataStore/Tests/DataStoreHostApp/AWSDataStorePluginLazyLoadTests/LL12/CompositePK/CompositePKChild.swift

```diff
@@ -34,7 +34,7 @@ public struct CompositePKChild: Model {
       self.createdAt = createdAt
       self.updatedAt = updatedAt
   }
-  public mutating func setParent(parent: CompositePKParent? = nil) {
+  public mutating func setParent(_ parent: CompositePKParent? = nil) {
```

Comment:
This is what should be codegenerated (see https://github.com/aws-amplify/amplify-codegen/pull/504/files#r1098000791) and is updated here manually.